<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotQClass</class>
 <widget class="QDialog" name="RobotQClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>小灵机器人语音</string>
  </property>
  <property name="windowIcon">
   <iconset resource="robotq.qrc">
    <normaloff>:/RobotQ/Resources/HIT_Robot_ico.ico</normaloff>:/RobotQ/Resources/HIT_Robot_ico.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPlainTextEdit" name="textStatus">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>56</y>
     <width>352</width>
     <height>146</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>380</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="robotq.qrc">:/RobotQ/Resources/博物馆语音交互背景.bmp</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuery">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>询问</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>291</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>开始对话</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStopspeak">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>291</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>停止说话</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comSpeaklist">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>黑龙江省博物馆开闭馆时间</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑龙江省博物馆有几层，都有哪些展厅</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑龙江省博物馆镇馆之宝有哪些？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>免费讲解</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>洗手间位于哪里？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>便民服务</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>文创天地</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>社会服务项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑龙江省博物馆有哪些活动？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑龙江省博物馆“相约龙博”课堂</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>环球自然日——青少年自然科学知识挑战赛</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>环球自然日——青少年科普绘画大赛</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑龙江省博物馆流动博物馆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑龙江省文博志愿者基地</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>博物馆简介</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnEnd">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>291</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>结束对话</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>textStatus</zorder>
  <zorder>btnQuery</zorder>
  <zorder>btnStart</zorder>
  <zorder>btnStopspeak</zorder>
  <zorder>comSpeaklist</zorder>
  <zorder>btnEnd</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="robotq.qrc"/>
 </resources>
 <connections/>
</ui>
